solutions/observability/cloud/monitor-cloudtrail-logs.md

```diff
@@ -209,11 +209,6 @@ Navigate to {{kib}} and choose among the following monitoring options:
     :alt: Visualize CloudTrail logs with Disocver
```

Comment:
The screenshot could be updated here to have the logs contextual experience enabled in Discover